CONFIGURATION.md

```diff
@@ -0,0 +1,90 @@
+# Configuration
+
+#### Creating a superuser
+
+*Is assumed that by this time all images are running on docker. (check: - [Running the application in local environment](#running-the-application-in-local-environment))
+
+1. Create a superuser for admin access running `python manage.py createsuperuser`.
+2. Go to `http://localhost:8010/admin` and log in with the superuser credentials.
+
+#### AWS Services Configuration in the Admin Web
+
+*In order to apply this configurations, login to the admin web is needed.
+
+1. **CloudFront Configuration:** Inside the Global configuration, apply the settings located in configuration/configuration.samples/cloud_front_configuration.sample on the CloudFront Configuration input.
+2. **MediaConvert Configuration:** Create a `MediaConvert Configuration` named `default`, using the settings found in `configuration/configuration.samples/media_convert_configuration.sample`.
+3. **MediaLive Configuration:** Create `MediaLive Configuration`, also named `default`, and apply the following configurations:
+   - **Input Configuration:** `configuration/configuration.samples/media_live_input_attachments.sample`.
+   - **Destination Configuration:** `configuration/configuration.samples/media_live_destinations.sample`.
+   - **Encoder Configuration:** `configuration/configuration.samples/media_live_encoder_settings_economic.sample`.
+
+   ![¡](docs/aws-services-configuration.png)
+
+#### Creating a Plan
+
+*On the "Organizations and channels" menu choose "Plans" and click on the "add plan" button.
+
+![¡](docs/orgs-and-channels.png)
+
+1. **Business name:** Add a name for the plan.
+2. **Video transcoding configuration:** From the dropdown choose the configuration needed.
```

Comment:
Dropdown is more a programmer expression

Reply:
also we did name things on previous step, it would be good to use those names, for example `From the selector choose the "default" configuration (or any other name you may have used)`. Apply this suggestion to other steps

Reply:
added.